composer.json

```diff
@@ -36,34 +36,36 @@
         "drupal/inline_entity_form": "~1.0-rc9",
         "drupal/json_field": "^1.0.0-rc2",
         "drupal/renderviz": "^1.0@alpha",
-        "drupal/styleguide": "~1.0-alpha3",
+        "drupal/styleguide": "~1.0@beta",
         "drupal/typed_link": "~1.1",
-        "drush/drush": "~9.0",
+        "drush/drush": "~10.3",
         "easyrdf/easyrdf": "1.0.0 as 0.9.1",
         "friends-of-behat/service-container-extension": "^1.0",
-        "guzzlehttp/guzzle": "~6.3",
         "instaclick/php-webdriver": "^1.4.7",
         "nikic/php-parser": "~3.0",
         "openeuropa/behat-transformation-context": "~0.1",
-        "openeuropa/code-review": "~1.5",
-        "openeuropa/drupal-core-require-dev": "^8.9.14",
-        "openeuropa/oe_content": "2.x-dev",
+        "openeuropa/code-review": "~1.6",
+        "openeuropa/drupal-core-require-dev": "^8.9 || ^9.1",
+        "openeuropa/oe_content": "dev-EWPP-911",
         "openeuropa/oe_corporate_blocks": "~4.0",
         "openeuropa/oe_corporate_countries": "~2.0",
         "openeuropa/oe_media": "~1.12",
         "openeuropa/oe_multilingual": "~1.8",
-        "openeuropa/oe_paragraphs": "dev-master",
-        "openeuropa/oe_search": "~1.0",
-        "openeuropa/oe_webtools": "~1.10",
-        "openeuropa/oe_contact_forms": "~1.0",
+        "openeuropa/oe_paragraphs": "dev-OEL-67",
+        "openeuropa/oe_search": "dev-OEL-60",
+        "openeuropa/oe_webtools": "~1.12",
+        "openeuropa/oe_contact_forms": "dev-OEL-164",
         "openeuropa/rdf_skos": "~1.0",
         "openeuropa/task-runner-drupal-project-symlink": "^1.0",
-        "phpunit/phpunit": "~6.0",
-        "symfony/dom-crawler": "~3.4"
+        "phpspec/prophecy-phpunit": "^1 || ^2",
+        "phpunit/phpunit": "^7.5 || ^8 || ^9"
```

Comment:
I've decided to freeze lowest version of phpunit on the 7.5 because this version introduced assertStringContainsString() that is used by tests/src/PatternAssertions/BasePatternAssert.php. We could use Drupal\Tests\PhpUnitCompatibilityTrait to bypass it, but unfortunately in Drupal 8 this class has name Drupal\Tests\PhpunitCompatibilityTrait so autoload can't find this file.